modules/ROOT/pages/feature-flagging.adoc

```diff
@@ -356,6 +356,22 @@ Suppressed errors are treated as underlying causes that can also be matched by O
 *Issue ID*
 
 * W-11855052
+
+<.^|`mule.forkJoin.completeChildContextsOnTimeout`
+|When enabled, the processors that perform fork and join work (currently, Scatter-Gather and Parallel For Each routers) complete the child event contexts when a timeout occurs.
```

Comment:
Is Scatter-Gather and Parallel For Each routers the exact name of the processors or should it be Scatter-Gather and Paralle for each router?